packages/ui-heading/src/Heading/README.md

```diff
@@ -50,12 +50,11 @@ example: true
 ```js
 ---
 example: true
-background: 'checkerboard-inverse'
```

Comment:
There are more examples like this in the docs app, if you do a project wide search for: `checkerboard-inverse` it will find it a couple places like: `Link, ProgessBar, ProgresCircle, Spinner, SVGIcon, Text`.

I think we have to change those as well just like this.